cpp/include/rapidsmpf/shuffler/shuffler.hpp

```diff
@@ -252,8 +252,8 @@ class Shuffler {
     rmm::cuda_stream_view stream_;
     BufferResource* br_;
     bool active_{true};
-    detail::PostBox inbox_;
-    detail::PostBox outbox_;
+    detail::PostBox<Rank> outgoing_chunks_;
+    detail::PostBox<PartID> received_chunks_;
```

Comment:
I would be good with some comments here that clearly explains the terminology of outgoing and receiving :)